### PR TITLE
Look and feel

### PR DESCRIPTION
**Pull request by markd315**
This changes the bootswatch theme.

### CODE REVIEW

**Review by kevdaci — DISMISSED** [review]
I know that user made a request for a change of theme, but we should improve it a little. We should probably start with the contrast of colors. Also the color scheme is not consistent.

**Comment by markd315** [comment]
@kevdaci Could you be more specific? I did change the contrast on the menu dropdowns already, what else?

**Comment by kevdaci** [comment]
@markd315 I do not think that the color of the menu bar goes well with the background color of website. Also, if you go to the Manage Module view, the blue, red, and orange, do not go well together. I think that the red buttons should be orange as well. Also, if you go to the create module view, the create button should be orange as well. I feel like @RyanClementsHax  should like at this too.

File: modules/core/client/views/header.client.view.html [hunk 1]
```diff
@@ -10,11 +10,13 @@
...
-        <a ng-switch-when="dropdown" class="dropdown-toggle" uib-dropdown-toggle role="button">{{::item.title}}&nbsp;<span class="caret"></span></a>
+      <li ng-repeat="item in vm.menu.items | orderBy: 'position'" ng-if="item.shouldRender(vm.authentication.user);" ng-switch="item.type" ng-class="{ dropdown: item.type === 'dropdown' }"
+       ui-sref-active="active" class="{{item.class}}" uib-dropdown="item.type === 'dropdown'">
+        <a ng-switch-when="dropdown" class="dropdown-toggle"
+        uib-dropdown-toggle role="button">{{::item.title}}&nbsp;<span class="caret"></span></a>
         <ul ng-switch-when="dropdown" class="dropdown-menu">
           <li ng-repeat="subitem in item.items | orderBy: 'position'" ng-if="subitem.shouldRender(vm.authentication.user);">
-            <a ui-sref="{{subitem.state}}({{subitem.params}})" ng-bind="subitem.title"></a>
+            <a ui-sref="{{subitem.state}}({{subitem.params}})" ng-bind="subitem.title" onMouseOver="this.style.color='#222222'" onMouseOut="this.style.color='#F0F0F0'"></a>
```
Comment (RyanClementsHax) [inline modules/core/client/views/header.client.view.html:19]: this is best done in css https://www.w3schools.com/cssref/sel_hover.asp

File: modules/core/server/views/layout.server.view.html [hunk 2]
```diff
@@ -16,7 +16,8 @@
 
   <!-- Application CSS Files -->
   {{#each cssFiles}}<link rel="stylesheet" href="/{{this}}">{{/each}}
-</head>
```
Comment (RyanClementsHax) [inline modules/core/server/views/layout.server.view.html:19]: you accidentally deleted the head closing tag

**Review by RyanClementsHax — DISMISSED** [review]

File: modules/core/client/views/header.client.view.html (hunk 1 above)
Comment (markd315) [reply modules/core/client/views/header.client.view.html:19]: I'll take a look at this after my meeting but yeah I kinda agree.

File: modules/core/server/views/layout.server.view.html (hunk 2 above)
Comment (markd315) [reply modules/core/server/views/layout.server.view.html:19]: That isn't showing on my local, it just got moved down a line with the bootstrap href tag.

**Review by kevdaci — APPROVED** [review]

**Pull request merged**